src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -969,6 +969,11 @@ OverlayImpl::send (protocol::TMValidation& m)
         if (! m.has_hops() || p->hopsAware())
             p->send(sm);
     });
+    STValidation::pointer val;
+    SerialIter sit (makeSlice(m.validation()));
+    val = std::make_shared <
+        STValidation> (std::ref (sit), false);
+    app_.getOPs().pubValidation (val);
```

Comment:
You many not need to create a Slice.

    SerialIter sit(
        m.validation().data(), m.validation().size());
    auto val = std::make_shared <
        STValidation> (std::ref (sit), false);